specification/securitycenter/resource-manager/Microsoft.Security/preview/2017-08-01-preview/informationProtection.json

```diff
@@ -0,0 +1,362 @@
+                    }
+                },
+                "tags": ["InformationProtectionPolicies"],
+                "description": "Details of the information protection policy.",
+                "operationId": "InformationProtectionPolicies_Get",
+                "parameters":
+                [{
+                        "$ref": "#/parameters/ApiVersion"
+                    }, {
+                        "$ref": "#/parameters/Scope"
+                    }, {
+                        "$ref": "#/parameters/InformationProtectionPolicyName"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/InformationProtectionPolicy"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "put": {
+                "x-ms-examples": {
+                    "Create or update an information protection policy for a management group": {
+                        "$ref": "./examples/InformationProtectionPolicies/CreateOrUpdateInformationProtectionPolicy_example.json"
+                    }
+                },
+                "tags": ["InformationProtectionPolicies"],
+                "description": "Details of the information protection policy.",
+                "operationId": "InformationProtectionPolicies_CreateOrUpdate",
+                "parameters": [{
+                        "$ref": "#/parameters/ApiVersion"
+                    }, {
+                        "$ref": "#/parameters/Scope"
+                    }, {
+                        "$ref": "#/parameters/InformationProtectionPolicyName"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/InformationProtectionPolicy"
+                        }
+                    },
+                    "201": {
+                        "description": "Created",
+                        "schema": {
+                            "$ref": "#/definitions/InformationProtectionPolicy"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
```

Comment:
Please check if this is a long running operation

Reply:
It's not a long running operation